examples/computer_vision/detr/data.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+"""
+COCO dataset which returns image_id for evaluation.
+Mostly copy-paste from https://github.com/pytorch/vision/blob/13b35ff/references/detection/coco_utils.py
+"""
+import os
+from io import BytesIO
+
+from google.cloud import storage
+from determined.util import download_gcs_blob_with_backoff
+
+import torchvision
+from PIL import Image
+
+from detr.util.misc import nested_tensor_from_tensor_list
+from detr.datasets.coco import ConvertCocoPolysToMask, make_coco_transforms
+
+
+def unwrap_collate_fn(batch):
+    batch = list(zip(*batch))
+    batch[0] = nested_tensor_from_tensor_list(batch[0])
+    batch[0] = {"tensors": batch[0].tensors, "mask": batch[0].mask}
+    return tuple(batch)
+
+
+class GCSBackend:
+    def __init__(self, bucket_name):
+        self._storage_client = storage.Client(project="determined-ai")
+        self._bucket = self._storage_client.bucket(bucket_name)
+
+    def convert_filepath(self, filepath):
+        tokens = filepath.split("/")
+        directory = tokens[-2]
+        filename = tokens[-1]
+        return "{}/{}".format(directory, filename)
+
+    def get(self, filepath):
+        filepath = self.convert_filepath(filepath)
+        blob = self._bucket.blob(filepath)
+        img_str = download_gcs_blob_with_backoff(blob)
+        return img_str
+
+
+class FakeBackend:
+    def __init__(self):
+        self.data = None
+
+    def get(self, filepath):
+        if self.data is None:
+            with open("imgs/train_curves.png", "rb") as f:
```

Comment:
I know this is generating fake data from the training curve, but can we maybe put a comment here just to be clear on that?